src/App.css

```diff
@@ -28,6 +28,11 @@
   color: #61dafb;
 }
 
+.container {
+  margin: 1rem;
+  text-align: center;
+}
+
 @keyframes App-logo-spin {
   from {
     transform: rotate(0deg);
```

Comment:
Hay que decidir si nos vamos a quedar con archivos css o si vamos a hacer todo css-in-js. Pero no hay que mezclar los dos.